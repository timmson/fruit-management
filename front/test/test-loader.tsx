import React from "react"
import renderer from "react-test-renderer"
import Loader from "../src/loader"

describe("Loader should", () => {

    test("return loader", () => {
        const component = renderer.create(<Loader/>)

        expect(component.toJSON()).toMatchSnapshot()

        component.unmount()
    })

})