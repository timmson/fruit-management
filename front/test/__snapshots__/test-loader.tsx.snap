// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should return loader 1`] = `
<p>
  Loading...
</p>
`;
